.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+# npx lint-staged
```

Comment:
I temporarily commented this out so I could commit and push.

Reply:
Our pre-commmit use  `npx lint-staged --relative`. When I check out this branch, that seems to work.

Reply:
Great! I'll get that change pushed up

Reply:
I think it's working now. I went ahead and fixed a couple files that the linter was complaining about, so we don't have to deal with them in the future.